spyderlib/utils/icon_manager.py

```diff
@@ -1,8 +1,16 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © 2015 The Spyder Development Team
```

Comment:
Pierre is part of the Team :-) So no, it's not a lie.

I've been changing all our copyrights like the line I mentioned above.